src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Action/Element/ActionElementInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Action\Element;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ * @author Oskar Stark <oskarstark@googlemail.com>
+ */
+interface ActionElementInterface
```

Comment:
This seems unused (there is one implementation no logic depending on this contract) and there is no explanation about its purpose. I still think it should be removed

Reply:
Agreed and removed 👍